test/erlang_vm_schema_tests.erl

```diff
@@ -28,8 +28,8 @@ basic_schema_test() ->
     cuttlefish_unit:assert_not_configured(Config, "vm_args.-kernel net_ticktime"),
     cuttlefish_unit:assert_not_configured(Config, "kernel.inet_dist_listen_min"),
     cuttlefish_unit:assert_not_configured(Config, "kernel.inet_dist_listen_max"),
-    case erlang:system_info(otp_release) of
-        [$R, $1, N|_] when N >= $6 ->
+    case cuttlefish:otp("R16", erlang:system_info(otp_release)) of
```

Comment:
Using the `cuttlefish:otp/2` function here is OK, provided the unit test for this function has asserted that the function is working correctly. As an alternative, the conditional code blocks could be swapped around and guarded by `[$R, $1, N|_] when N =< $5`